peachjam/templates/peachjam/gazette_year.html

```diff
@@ -12,16 +12,25 @@
             <li class="breadcrumb-item">
               <a href="{% url 'gazettes' %}">{% trans 'Gazettes' %}</a>
             </li>
+            {% if country %}
+              <li class="breadcrumb-item">
+                <a href="{% url 'gazettes_by_locality' country.pk.lower %}">{{ country }}</a>
+              </li>
+            {% endif %}
             {% if locality %}
               <li class="breadcrumb-item">
-                <a href="{% url 'gazettes_by_locality' code=locality.code %}">{{ locality }}</a>
+                <a href="{% url 'gazettes_by_locality' locality.place_code %}">{{ locality }}</a>
               </li>
             {% endif %}
+            <li class="breadcrumb-item active">{{ year }}</li>
           </ol>
         </nav>
       {% endblock %}
       {% block page-title %}
-        <h1 class="mt-4">{% trans 'Gazettes' %} - {{ year }}</h1>
+        <h1 class="mt-4">
+          {{ place }}
```

Comment:
```suggestion
          {{ place|default_if_none:"" }}
```

otherwise when we go straight to the year view without choosing a place we get this:
![image](https://github.com/laws-africa/peachjam/assets/25079238/05aac2ab-a146-4120-9b80-db7100bdd195)

Reply:
same with page title above

Reply:
I can't repro this, what's the URL? 
![image](https://github.com/laws-africa/peachjam/assets/4178542/8d17cc04-f743-4eb5-a467-859770478642)

Reply:
LOL it's because I committed your fix